client/src/pages/Profile/Profile.tsx

```diff
@@ -2,43 +2,57 @@ import { Grid, List, ListItem, ListItemText, Paper } from "@material-ui/core";
 import useStyles from './useStyles';
 import { useState } from 'react';
 import MainPanel from '../../components/ProfileSettings/MainPanel/MainPanel';
+import { BrowserRouter as Router, Switch, Route, Link, useRouteMatch } from 'react-router-dom';
+import { v4 as uuid } from 'uuid';
 
 export default function Profile(): JSX.Element {
     const classes = useStyles();
 
+    const { path, url } = useRouteMatch();
     const [ compIndex, setIndex ] = useState(0);
 
     const listText = [
-        "Edit Profile",
-        "Profile photo",
-        "Availability",
-        "Payment",
-        "Security",
-        "Settings",
+        { link: 'editProfile', text:"Edit Profile" },
```

Comment:
Please use hyphenated case names for route with multiple words.

Eg: `main-page`